evalml/model_understanding/graphs.py

```diff
@@ -818,6 +876,10 @@ def graph_partial_dependence(
         ]
         if any(is_categorical):
             features = _put_categorical_feature_first(features, is_categorical[0])
+        if kind == "individual" or kind == "both":
+            raise ValueError(
+                "Individual conditional expectation plot can only be created with a one-way partial dependence plot"
+            )
```

Comment:
Currently, I'm leaving out the ICE plot when plotting a two-way PD. I think the current behavior of raising an error is the best move, although I've considered putting in a warning message in the log. Open to suggestions for this behavior 🙂